src/features/venueMap/pages/modals/VenueTypeModal/VenueTypeModal.native.test.tsx

```diff
@@ -70,6 +71,32 @@ describe('<VenueTypeModal />', () => {
       })
     })
 
+    it('should trigger ApplyVenueMapFilter log when pressing search button and venue type code selected', async () => {
+      render(<VenueTypeModal hideModal={mockHideModal} isVisible />)
+
+      fireEvent.press(screen.getByText('Cinéma - Salle de projections'))
+
+      fireEvent.press(screen.getByText('Rechercher'))
+
+      await waitFor(() => {
+        expect(analytics.logApplyVenueMapFilter).toHaveBeenNthCalledWith(1, {
+          venueType: 'Cinéma - Salle de projections',
+        })
+      })
+    })
+
+    it('should not trigger ApplyVenueMapFilter log when pressing search button and venue type code is "Tout"', async () => {
+      render(<VenueTypeModal hideModal={mockHideModal} isVisible />)
+
+      fireEvent.press(screen.getByText('Tout'))
+
+      await act(async () => {
```

Comment:
Il y a une raison particulière pour wrapper ton fireEvent dans un act ? A priori tu n'as pas besoin : https://testing-library.com/docs/preact-testing-library/api/#fireevent

Reply:
J'ai une erreur sans act, ça passe avec un waitFor mais on est sur un not (un not pas visuel cela dit). Vu qu'on est sur une modal il y a un petit temps avant qu'elle se ferme.